src/SmartComponents/Inventory/applications/advisor/Advisor.js

```diff
@@ -0,0 +1,126 @@
+        const { entity } = this.props;
+        try {
+            const inventoryReport = (await fetch(`${SYSTEM_FETCH_URL}${entity.id}/reports`)).data;
+            this.setState({
+                inventoryReport,
+                inventoryReportFetchStatus: 'fulfilled'
+            });
+        } catch (error) {
+            this.props.addNotification({
+                variant: 'danger',
+                dismissable: true,
+                title: '',
+                description: 'Inventory item rule fetch failed.'
+            });
+            this.setState({
+                inventoryReportFetchStatus: 'failed'
+            });
+        }
+    }
+
+    expandAll(expanded) {
+        this.setState({ expanded: !expanded });
+    }
+
+    buildRuleCards = () => {
+        const { inventoryReport, expanded } = this.state;
+        return (
+            <Fragment>
+                <div className="pf-u-display-flex pf-u-flex-direction-row-reverse">
+                    <a onClick={ e => {
+                        e.preventDefault();
+                        this.expandAll(expanded);
+                    } } rel="noopener">
+                        { (expanded ? `Collapse All` : `Expand All`) }
+                    </a>
+                </div>
+                {
+                    inventoryReport.active_reports.map((report, key) =>
+                        <ExpandableRulesCard key={ key } report={ report } isExpanded={ expanded } />
+                    ) }
+            </Fragment>
+        );
+    }
+
+    render() {
+        const { inventoryReport, inventoryReportFetchStatus } = this.state;
+        return (
+            <Fragment>
+                { inventoryReportFetchStatus === 'pending' && (
+                    <Card>
+                        <CardBody>
+                            <List />
+                        </CardBody>
+                    </Card>
+                ) }
+                { inventoryReportFetchStatus === 'fulfilled' && (
+                    inventoryReport ? this.buildRuleCards() :
+                        <Card className="ins-empty-rule-cards">
+                            <CardHeader>
+                                <CommentSlashIcon size='lg' />
+                            </CardHeader>
+                            <CardBody>
+                                No data available for configuration assessment at the moment.
+                            </CardBody>
+                        </Card>
+                ) }
+                { inventoryReportFetchStatus === 'failed' && (
+                    <Card className="ins-empty-rule-cards">
+                        <CardHeader>
+                            <FrownOpenIcon size='lg' />
+                        </CardHeader>
+                        <CardBody>
+                            There was an error while trying to fetch rules list for this entity. Please contact your
+                            administrator and show him this screen.
```

Comment:
Gender bias alarms going off, unless this is a uxd sanctioned message would recommend considering the following rewording: 

" There was an error fetching rules list for this Entity. Please show your administrator this screen."